### PR TITLE
Adding datacard for Wiki-Auto

### PR DESCRIPTION
**Pull request by ddhruvkr**
Adding a first draft of the Wiki-Auto data card.

### CODE REVIEW

File: web/tasks/Wiki-Auto.md [hunk 1]
```diff
@@ -1,119 +1,213 @@
...
 ## Dataset Creation
 
 ### Curation Rationale
 
-[More Information Needed]
+Wiki-Auto provides a new version of the Wikipedia corpus that is larger, contains 75% less defective pairs and has more complex rewrites than the previous WIKILARGE dataset.  
 
 ### Communicative Goal
 
-[More Information Needed]
```
Comment (sebastianGehrmann) [inline web/tasks/Wiki-Auto.md:68]: Can you add something here? It'd be sufficient to write something like "The goal is to communicate the same information as the source sentence in an easier-to-understand manner" or so.

File: web/tasks/Wiki-Auto.md [hunk 2]
```diff
@@ -1,119 +1,213 @@
...
 
 #### Annotation process
 
-[More Information Needed]
+Sentence alignment labels were crowdsourced for 500 randomly sampled document pairs (10,123 sentence pairs total). The authors pre-selected several alignment candidates from English Wikipedia for each Simple Wikipedia sentence based on various similarity metrics, then asked the crowd-workers to annotate these pairs. Finally, they trained their alignment model on this manually annotated dataset to obtain automatically aligned sentences (138,095 document pairs, 488,332 sentence pairs).
 
 #### Who are the annotators?
 
-[More Information Needed]
+No demographic annotation is provided for the crowd workers.
```
Comment (sebastianGehrmann) [inline web/tasks/Wiki-Auto.md:183]: Is it mentioned what platform was used to annotate?

File: web/tasks/Wiki-Auto.md [hunk 3]
```diff
@@ -1,119 +1,213 @@
...
 
 ### Other Known Limitations
 
-[More Information Needed]
+Since the data is created using an automatic alignment model (which is not perfect) there could be still some alignment issues in the data.
 
 ## Getting started with in-depth research on the task
 
-[More Information Needed]
+The dataset can be downloaded from the original repository [(here)](https://github.com/chaojiang06/wiki-auto) by the authors (it also contains instructions to load a trained model on the dataset) or can also be used via HuggingFace and TFDS.
```
Comment (sebastianGehrmann) [inline web/tasks/Wiki-Auto.md:213]: Can you add links to the HF / TFDS mentions here? 

And maybe link papers that describe common approaches that should be used as baselines for people interested in working on the task.

File: web/tasks/Wiki-Auto.md (hunk 2 above)
Comment (ddhruvkr) [reply web/tasks/Wiki-Auto.md]: Added the platform used.

File: web/tasks/Wiki-Auto.md (hunk 3 above)
Comment (ddhruvkr) [reply web/tasks/Wiki-Auto.md]: Added link to the HF and TFDS (for TFDS the PR is not merged yet, therefore added the link to the tfds homepage. Will update once it is merged). Also added the recent models proposed for simplification.

**Comment by sebastianGehrmann** [comment]
Fantastic, thank you!

**Pull request merged**